extensions/wikia/PhalanxII/classes/PhalanxPager.class.php

```diff
@@ -75,83 +80,82 @@ function getIndexField() {
 	}
 
 	function getStartBody() {
-		return Html::openElement( 'ul', array( "id" => 'phalanx-block-' . $this->pInx . '-' . $this->id . '-stats' ) );
+		return Html::openElement( 'ul', [ "id" => 'phalanx-block-' . $this->pInx . '-' . $this->id . '-stats' ] );
 	}
 
 	function getEndBody() {
 		return Html::closeElement( 'ul' );
 	}
 
 	function getEmptyBody() {
-		return Html::element('div', array('class' => 'error'), wfMsg('phalanx-no-results'));
+		return Html::element( 'div', [ 'class' => 'error' ], $this->msg( 'phalanx-no-results' )->text() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "class" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)